src/index.js

```diff
@@ -88,7 +88,8 @@ const toggleThrottle = () => {
   render();
 };
 
-const throttleRateChange = (kBps) => {
+const throttleRateChange = (event) => {
+  const kBps = parseInt(event.target.value, 10);
```

Comment:
`event.target.value` should be contained within the component it came from, `throttleRateChange` shouldn't have to worry about it. [Example](https://github.com/tomitm/james/blob/new-mapping/src/component/mapping/new-mapping-target.js#L8)